charts/linkerd2/templates/proxy-injector-rbac.yaml

```diff
@@ -89,7 +91,10 @@ webhooks:
       name: linkerd-proxy-injector
       namespace: {{ .Values.global.namespace }}
       path: "/"
-    caBundle: {{ ternary (b64enc $ca.Cert) (b64enc (trim .Values.proxyInjector.crtPEM)) (empty .Values.proxyInjector.crtPEM) }}
+{{- if and (.Values.proxyInjector.externalSecret) (empty .Values.proxyInjector.caBundle) }}
+  {{- fail "If proxyInjector.externalSecret is true then you need to provide proxyInject.caBundle" }}
```

Comment:
This should be `proxyInjector.caBundle`, right?

Reply:
yes